### PR TITLE
feat(route/bilibili): refactor description with art-template

### PR DESCRIPTION
**Pull request by nuomi1**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/bilibili/bangumi/media/9192
/bilibili/precious
/bilibili/user/coin/208259
/bilibili/fav/756508/50948568
/bilibili/user/like/208259
/bilibili/video/page/BV1i7411M7N9
/bilibili/partion/ranking/171/3
/bilibili/partion/33
/bilibili/popular/all
/bilibili/ranking/0/3/1
/bilibili/user/channel/2267573/396050
/bilibili/user/collection/245645656/529166
/bilibili/user/fav/2267573
/bilibili/weekly
/bilibili/user/dynamic/2267573
/bilibili/followings/dynamic/2267573
/bilibili/followings/video/2267573
/bilibili/user/video-all/2267573
/bilibili/user/video/2267573
/bilibili/vsearch/RSSHub
/bilibili/watchlater/2267573
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

1. refactor bangumi route with ofetch and art-template
2. refactor other routes with art-template
3. use embed instead of disableEmbed

### CODE REVIEW

File: lib/routes/bilibili/bangumi.ts [hunk 1]
```diff
@@ -1,66 +1,68 @@
...
-            method: 'get',
-            url: `https://www.bilibili.com/bangumi/media/md${mediaid}`,
-        });
-        mediaData = JSON.parse(response.data.match(/window\.__INITIAL_STATE__=([\S\s]+);\(function\(\)/)[1]) || {};
-        seasonid = mediaData.mediaInfo.season_id;
-    }
-    const { data } = await got.get(`https://api.bilibili.com/pgc/web/season/section?season_id=${seasonid}`);
+    const mediadata = await utils.getBangumi(mediaid, cache);
+    const seasonid = String(mediadata.season_id);
+    const seasondata = await utils.getBangumiItems(seasonid, cache);
```
Comment (TonyRL) [inline lib/routes/bilibili/bangumi.ts:35]: [`camelCase`](https://docs.rsshub.app/joinus/advanced/script-standard#naming)

File: lib/routes/bilibili/bangumi.ts [hunk 2]
```diff
@@ -1,66 +1,68 @@
...
+import { EpisodeResult } from './types';
+import utils from './utils';
 
 export const route: Route = {
-    path: '/bangumi/media/:mediaid',
+    path: '/bangumi/media/:mediaid/:embed?',
     name: '番剧',
+    maintainers: ['DIYgod', 'nuomi1'],
+    handler,
+    example: '/bilibili/bangumi/media/9192',
```
Comment (TonyRL) [inline lib/routes/bilibili/bangumi.ts:11]: Please keep these properties in their original order. Do not perform unnecessary reordering.

File: lib/routes/bilibili/templates/description.art [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{{ if embed }}
+{{ if ugc }}
+<iframe width="640" height="360" src="https://player.bilibili.com/player.html?aid={{ aid }}&cid={{ cid }}&bvid={{ bvid }}" frameborder="0" allowfullscreen></iframe>
+{{ /if }}
+{{ if ogv }}
+<iframe width="640" height="360" src="https://player.bilibili.com/player.html?seasonId={{ seasonId }}&episodeId={{ episodeId }}" frameborder="0" allowfullscreen></iframe>
```
Comment (TonyRL) [inline lib/routes/bilibili/templates/description.art:6]: This iframe brings back the issue that is fixed in #14517

File: lib/routes/bilibili/bangumi.ts (hunk 1 above)
Comment (nuomi1) [reply lib/routes/bilibili/bangumi.ts]: Done.

File: lib/routes/bilibili/bangumi.ts (hunk 2 above)
Comment (nuomi1) [reply lib/routes/bilibili/bangumi.ts]: Done.

File: lib/routes/bilibili/templates/description.art (hunk 3 above)
Comment (nuomi1) [reply lib/routes/bilibili/templates/description.art]: Done.

**Pull request merged**